src/deb/init.d/jmxtrans

```diff
@@ -1,30 +1,30 @@
 #!/bin/bash
 #
-# /etc/init.d/@PACKAGE -- startup script for @PACKAGE
+# /etc/init.d/jmxtrans -- startup script for jmxtrans
```

Comment:
Yes, the history here is that I created a ton (25+) debian installers for all of marinsoftware's internal projects. This was just one more project internally. So, I aliased and reused as much as possible. It is ok to remove that here now.